sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -651,7 +668,9 @@ int64_t CurlSession::OnRead(Context const& context, uint8_t* buffer, int64_t cou
 
     if (this->IsEOF())
     {
-      // Final CRLF after end of chunk
+      /* For a chunk response, EOF means that the last chunk 0 was found.
```

Comment:
What's a `last chunk 0`?

```suggestion
      /* For a chunk response, EOF means that the last chunk was found.
```